app/index.js

```diff
@@ -207,8 +208,12 @@ module.exports = yeoman.generators.Base.extend({
   writing: {
     app: function() {
       var self = this;
-      var files = glob.sync(self.templatePath() + '/**/*');
-
+	  if(!win32){
+        var files = glob.sync(self.templatePath + '/**/*');
+	  } else {
+		var files = glob.sync(self.templatePath + '\**\*');
```

Comment:
I'm not sure about this given [this](https://github.com/isaacs/node-glob#windows)